docs/spec.md

```diff
@@ -81,7 +81,7 @@ completely numeric to simplify generation of StableHLO programs.
 
 ```ebnf
 Type         ::= ValueType | NonValueType
-ValueType    ::= TensorType | TokenType | TupleType
+ValueType    ::= TensorType | QuantizedTensorType | TokenType | TupleType
```

Comment:
Should the QuantizedTensorType be a special case of TensorType instead of ValueType?

For example, quantized tensor constants can not be declared since it is a part of TensorType in the spec.

Reply:
Incidentally, this is what the initial version of the pull request did, but we decided to split TensorType and QuantizedTensorType recently, for the reasons described in https://github.com/sdasgup3/stablehlo/pull/5#discussion_r1161141397.